lib/navigation_tab.dart

```diff
@@ -0,0 +1,29 @@
+
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:gocast_mobile/views/course_view/downloaded_courses_view/downloaded_courses_view.dart';
+import 'package:gocast_mobile/views/course_view/pinned_courses_view/pinned_courses_view.dart';
+import 'package:gocast_mobile/views/notifications_view/notifications_overview.dart';
+import 'providers.dart';
```

Comment:
Also line 7 and 8

Reply:
As I said, I will make another PR for clean code, so that all such things we can address there. 
There are many things in other files also. But if you want specifically for this, i can upload